#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_memset_include_cstring.dpatch by Krzysztof Burghardt <krzysztof@burghardt.pl>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes error: ‘memset’ is not a member of ‘std’.

@DPATCH@
diff -urNad tiemu-3.02~/src/ipc/dcop/dcop.cpp tiemu-3.02/src/ipc/dcop/dcop.cpp
--- tiemu-3.02~/src/ipc/dcop/dcop.cpp	2007-12-14 22:28:08.000000000 +0100
+++ tiemu-3.02/src/ipc/dcop/dcop.cpp	2009-08-09 23:36:14.000528344 +0200
@@ -18,6 +18,7 @@
  */
 
 #include <cstdlib>
+#include <cstring>
 #include <glib.h>
 #include <gtk/gtk.h>
 #include <dcopobject.h>
